ngraph_bridge/ngraph_builder.cc

```diff
@@ -1977,13 +1977,27 @@ static Status TranslateFillOp(
 static Status TranslateFloorDivOp(
     const Node* op, const std::vector<const Tensor*>& static_input_map,
     Builder::OpMap& ng_op_map) {
-  auto ng_floordiv = [&op](std::shared_ptr<ng::Node> ng_input1,
-                           std::shared_ptr<ng::Node> ng_input2) {
-    return ConstructNgNode<ng::op::Floor>(
-        op->name(),
-        ConstructNgNode<ng::op::Divide>(op->name(), ng_input1, ng_input2));
-  };
-  return TranslateBinaryOp(op, static_input_map, ng_op_map, ng_floordiv);
+  DataType dtype;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "T", &dtype));
+  std::initializer_list<DataType> int_types = {DT_INT8, DT_UINT16, DT_INT16,
```

Comment:
you can create Int types in utils so that it can be reused for some other op
https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_utils.cc#L354